packages/color-block/src/helpers/mapColorSpaces.ts

```diff
@@ -0,0 +1,168 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { FrontifyColor } from '@frontify/app-bridge';
+
+import { ColorSpaceInputValues } from '../types';
+
+type ColorSpace = {
+    id: string;
+    key?: string;
+    label: string;
+    value?: string;
+    placeholder?: string;
+};
+
+export const mapColorSpaces = (colorSpaceId: keyof ColorSpaceInputValues, color?: FrontifyColor): ColorSpace => {
+    switch (colorSpaceId) {
+        case 'hex':
+            return {
+                id: colorSpaceId,
+                label: 'HEX',
```

Comment:
here is the same with the naming of the label. Maybe use a type or enum. could you combine it with the `export type ColorSpaceInputValues = {` within the helper file? wdyt?